amarachukwu-agbo/random-number-generator
# #165145166 Add endpoint to delete phone numbers

#### What does this PR do?
Add endpoint to delete phone numbers

#### Description of Task to be completed?
- Add route for endpoint
- Add controller for endpoint
- Write tests

#### How should this be manually tested?
- Clone the repo and install application dependencies using `yarn`. 
- Run command  `yarn server:start-dev`.
- Using Postman, visit the url `DELETE http://localhost:3000/api/v1/numbers`, you should see a message informing that numbers have been deleted.
- Run `yarn test` to test the feature

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145166](https://www.pivotaltracker.com/story/show/165145166)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
